psh/pshapp/pshapp.c

```diff
@@ -1771,7 +1772,7 @@ void psh_pshappexitinfo(void)
 int psh_pshapp(int argc, char **argv)
 {
 	char *path = NULL;
-	const char *consolePath = _PATH_CONSOLE;
+	psh_consolePath = _PATH_CONSOLE;
```

Comment:
or `static const char *` here

Reply:
we need to access `psh_consolePath` in `dmesg.c`